src/sonic-config-engine/setup.py

```diff
@@ -19,4 +20,7 @@ def get_test_suite():
       scripts=['sonic-cfggen'],
       install_requires=['lxml', 'jinja2', 'netaddr', 'ipaddr', 'pyyaml', 'pyangbind'],
       test_suite='setup.get_test_suite',
+      data_files=[
+        ('/usr/share/sonic/templates', glob.glob('data/*')),
+      ],
      )
```

Comment:
Why do we install this file with sonic-config-engine? Shouldn't it go with base image?